### ISSUE TITLE
Add support for configuring which attribute to look for the email under ldap

### ISSUE THREAD

**Issue by paladox** [Graylog2/graylog2-server#11131]
## Expected Behavior
You should be able to configure the attribute to look for the email under ldap.

## Current Behavior
I presume it looks for a default email attribute or maybe doesn't even look for one?

It's currently defaulting to unknown@unknown.invalid.

## Steps to Reproduce (for bugs)

1. Setup ldap
2. Setup ldap configuation
3. Login as a user in ldap
4. View /system/users and look at the email column.

## Your Environment

* Graylog Version: 4.1.2+20cd592
* Java Version: openjdk 11.0.12 2021-07-20
* Elasticsearch Version: 7.13.4
* MongoDB Version: 4.4.7 
* Operating System: debian 10
* Browser version: 92.0.4515.131

**Comment by kroepke** [comment]
Yes, this is not configurable at the moment, Graylog looks for `mail` and `rfc822Mailbox` in that order right now:
https://github.com/Graylog2/graylog2-server/blob/f1b883bc4a0686156640901caba97b811f0092ed/graylog2-server/src/main/java/org/graylog/security/authservice/ldap/UnboundLDAPConnector.java#L296

We'll look at making this configurable in a future version.
Thanks!
